icrc-84.md

```diff
@@ -233,24 +231,19 @@ then it may want to also expand the response record with a field describing the
 ## Tracked balance
 
 It was said above that `deposit_inc` returned by `notify` is the difference in deposit balance relative to the last known (= "tracked") deposit balance.
-The tracked deposit balance can be queried with the following method.
+The tracked deposit balance can be queried alongside with the credit with the following method.
 
 ```candid "Methods" +=
-icrc84_trackedDeposit : (Token) -> (BalanceResult) query;
+icrc84_query : (vec Token) -> (vec record {
+    Token;
+    record {
+        credit : int;
+        tracked_deposit : opt Amount;
+    };
+}) query;
```

Comment:
This is the same method that we cited above already. We don't have to repeat the function.